src/main/webapp/app/exam/manage/exams/exam-update.component.ts

```diff
@@ -200,7 +217,7 @@ export class ExamUpdateComponent implements OnInit {
      * Validates the EndDate inputted by the user.
      */
     get isValidEndDate(): boolean {
-        return this.exam.endDate !== undefined && dayjs(this.exam.endDate).isAfter(this.exam.startDate);
+        return this.exam.endDate !== undefined && this.exam.endDate !== null && dayjs(this.exam.endDate).isAfter(this.exam.startDate);
```

Comment:
same for the other cases below.
```suggestion
        return !!this.exam.endDate && dayjs(this.exam.endDate).isAfter(this.exam.startDate);
```